jigasi/rootfs/defaults/sip-communicator.properties

```diff
@@ -187,3 +191,24 @@ org.jitsi.jigasi.MUC_SERVICE_ADDRESS={{ $XMPP_MUC_DOMAIN }}
 {{ if $TRUSTED_DOMAIN_LIST }}
 org.jitsi.jigasi.TRUSTED_DOMAINS=[ {{ range $index, $element := $TRUSTED_DOMAINS }}{{ if gt $index 0 }},{{ end }}"{{ $element }}"{{ end}} ]
 {{ end }}
+
+org.jitsi.jigasi.JVB_INVITE_TIMEOUT={{ $JIGASI_JVB_TIMEOUT }}
+
+{{ if $JIGASI_LOCAL_REGION }}
+org.jitsi.jigasi.LOCAL_REGION={{ $JIGASI_LOCAL_REGION }}
+{{ end }}
+
+{{ if $BOSH_URL_PATTERN }}
+org.jitsi.jigasi.xmpp.acc.BOSH_URL_PATTERN={{ $BOSH_URL_PATTERN }}
+{{ end }}
+
+{{ if $USE_TRANSLATOR_IN_CONFERENCE }}
+org.jitsi.jigasi.xmpp.acc.USE_TRANSLATOR_IN_CONFERENCE=true
+net.java.sip.communicator.impl.protocol.sip.acc1.USE_TRANSLATOR_IN_CONFERENCE=true
+# Should be enabled when using translator mode
+net.java.sip.communicator.impl.neomedia.audioSystem.audiosilence.captureDevice_list=["AudioSilenceCaptureDevice:noTransferData"]
+{{ end }}
+
+{{ if .Env.JIGASI_CONFIGURATION -}}
```

Comment:
What is this?

Reply:
Same thing as https://github.com/jitsi/docker-jitsi-meet/blob/master/prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua#L267 - we are setting some more custom options to finetune Jigasi in our setup, like SIP_PREFERRED_CLEAR_PORT but i thought it wouldnt make sense to expose every possible config option seperated, so JIGASI_CONFIGURATION can be used to render custom snippets inside the config in addition to the predefined options..